<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="filePathLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="classifyButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Classify This</string>
    </property>
   </widget>
   <widget class="QToolButton" name="loadFileButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>24</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Image file:</string>
    </property>
   </widget>
   <widget class="QLabel" name="answerLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>741</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Answer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="imageGraphicsView">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>501</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
